src/DevBetterWeb.Core/Specs/InvitationByInviteCodeWithSubscriptionIdSpec.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Ardalis.Specification;
+using DevBetterWeb.Core.Entities;
+
+namespace DevBetterWeb.Core.Specs
+{
+  public class InvitationByInviteCodeWithSubscriptionIdSpec : Specification<Invitation>
+  {
+    public InvitationByInviteCodeWithSubscriptionIdSpec(string inviteCode)
+    {
+      Query.Where(invite => invite.InviteCode == inviteCode);
+      Query.Include(invite => invite.PaymentHandlerSubscriptionId);
```

Comment:
No need to .Include primitive types on the entity. Just navigation properties to other entities.